apple/internal/resource_actions/actool.bzl

```
@@ -208,12 +239,24 @@ def compile_asset_catalog(
         platform_prerequisites.device_families,
     ))
 
-    outputs = [output_dir]
+    alticons_outputs = []
+    actool_output_plist = None
+    actool_outputs = [output_dir]
     if output_plist:
-        outputs.append(output_plist)
+        if alternate_icons:
+            alticons_outputs = [output_plist]
+            actool_output_plist= intermediates.file(
```

Formatting (not sure why buildifier can’t catch this).
```suggestion
            actool_output_plist = intermediates.file(
```

---

Not my proudest moment